.github/workflows/ci.yml

```
@@ -49,7 +49,7 @@ jobs:
         with:
           name: code-coverage-ubuntu-latest-20.x
           path: coverage/
-      - uses: SonarSource/sonarcloud-github-action@v2.2.0
+      - uses: SonarSource/sonarcloud-github-action@v2.3.0
```

```suggestion
      - uses: SonarSource/sonarcloud-github-action@v2
```

It's better to use the `v2` tag for this